Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml

```
@@ -325,9 +325,9 @@
   description: Rather bland and inoffensive. Perfect for vanishing off the face of the universe.
   components:
   - type: Sprite
-    sprite: Clothing/Uniforms/Jumpsuit/hop.rsi
+    sprite: _Harmony/Clothing/Uniforms/Jumpsuit/hop.rsi
```

```suggestion
    sprite: _Harmony/Clothing/Uniforms/Jumpsuit/hop.rsi # Harmony change. Resprite
```
Upstream file changes require a comment.